ci/upload-test-metrics.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -euo pipefail
+
+# Fetch the test summary artifacts uploaded earlier
+mkdir "./test_summaries"
+buildkite-agent artifact download "*test_summary*.json" "./test_summaries"
+
+# Fetch Google credentials so that we can upload the metrics to the GCS bucket.
+GCP_SECRET="$(mktemp)"
+imp-ci secrets read --environment=production --buildkite-org=improbable \
+    --secret-type=gce-key-pair --secret-name=qa-unreal-gce-service-account \
+    --write-to=${GCP_SECRET}
+gcloud auth activate-service-account --key-file "${GCP_SECRET}"
+
+# Define target upload location
+PROJECT="holocentroid-aimful-6523579"
+DATASET="UnrealGDK"
+TABLE="ci_metrics"
+
+for json_file in ./test_summaries/*.json; do
+    cat $json_file | bq --project_id "$PROJECT" --dataset_id "$DATASET" insert "$TABLE"
```

Comment:
```suggestion
    cat "${json_file}" | bq --project_id "${PROJECT}" --dataset_id "{$DATASET}" insert "${TABLE}"
```

Reply:
For consistency + to handle the case where `json_file` has spaces

Reply:
Yup, good one - fixed it now.